.github/workflows/ci.yaml

```diff
@@ -98,9 +98,36 @@ jobs:
         run: ./scripts/get-baton.sh && mv baton /usr/local/bin
       - name: Import sql into postgres
         run: |
-          psql -h localhost --user bitbucket -f schema.sql
           psql -h localhost --user bitbucket -f data.sql
         env:
           PGPASSWORD: ${{ secrets.DB_PASSWORD }}
       - name: Build baton-bitbucket-datacenter
         run: go build ./cmd/baton-bitbucket-datacenter
+      - name: Check status
+        run: |
+          curl -LI localhost:7990/status -o /dev/null -w '%{http_code}\n' -s        
+          sleep 180s
```

Comment:
It does not fails. It just shows the error code. Github actions does not handle the same time for setting the bitbucket app up. Because of that, a sleep time is necessary. Otherwise, those tests runs before the bitbucket api responds properly.